cishouseholds/pipeline/validation_schema.py

```diff
@@ -376,6 +376,19 @@
         "ctry12": string_dict,
         "ctry_name12": string_dict,
         "tranche": integer_dict,
+        "cis20_samp": string_dict,
+        "cis20_name": string_dict,
+        "preference": string_dict,
+        "ons_household_id": integer_dict,
```

Comment:
Please change this to a string, otherwise leading 0's will be removed from the ID